samplers/aws/xray/remote_sampler.go

```diff
@@ -80,18 +83,46 @@ func NewRemoteSampler(ctx context.Context, opts ...Option) (*RemoteSampler, erro
 		clientID:                     id,
 		xrayClient:                   newClient(cfg.proxyEndpoint),
 		samplingRulesPollingInterval: cfg.samplingRulesPollingInterval,
+		fallbackSampler:              NewFallbackSampler(),
 	}
 
-	// starts the rule poller
+	// starts the rule and target poller
 	remoteSampler.start(ctx)
 
 	return remoteSampler, nil
 }
 
 func (rs *RemoteSampler) ShouldSample(parameters sdktrace.SamplingParameters) sdktrace.SamplingResult {
-	// ToDo: add business logic for remote sampling
+	// Use fallback sampler with sampling config 1 req/sec and 5% of additional requests if manifest is expired
+	if rs.manifest.expired() {
+		globalLogger.Printf("Centralized manifest expired. Using fallback sampling strategy")
+		return rs.fallbackSampler.ShouldSample(parameters)
+	}
+
+	rs.manifest.mu.RLock()
+	defer rs.manifest.mu.RUnlock()
+
+	// Match against known rules
+	for _, r := range rs.manifest.rules {
 
-	return sdktrace.SamplingResult{}
+		r.mu.RLock()
+		applicable := r.AppliesTo()
+		r.mu.RUnlock()
+
+		if !applicable {
+			continue
+		}
+
+		globalLogger.Printf("Applicable rule: %s", *r.ruleProperties.RuleName)
+
+		samplingResult := r.Sample(parameters)
+
+		return samplingResult
```

Comment:
```suggestion
		if applicable {
			globalLogger.Printf("Applicable rule: %s", *r.ruleProperties.RuleName)
			return r.Sample(parameters)
		}
```

This can be made more concise by inverting the condition.  Again, logging is probably not appropriate here.